submit-api/src/submit_api/resources/staff/submitted_document.py

```diff
@@ -62,3 +62,25 @@ def get():
 
         documents = DocumentService.get_all_documents(search_options)
         return SubmittedDocumentByProjectSchema(many=True).dump(documents), HTTPStatus.OK
+
+
+@cors_preflight("GET, OPTIONS")
```

Comment:
i would probably make a generic api with a status filter here unless failed submission are totally a different deal
like item/:item_id?status=failed

Reply:
I had a similar thought. But I thought this endpoint is akin to check if this item has failed in the past. And I just needed that and didn't need a generic endpoint. I thought we could always refactor if we needed.